packages/components/datetime/DateTime.mdx

```diff
@@ -28,19 +28,22 @@ import { DateTime } from '@contentful/f36-datetime';
 
 ## Examples
 
-### Basic usage
-
-All the usages below will result in 'Tue, 17 Aug 2021 at 3:45 PM'
-
-```tsx
-<Stack flexDirection="column">
-  {/* passing a Timestamp as an ISOString */}
-  <DateTime date="2021-08-17T15:45" />
-  {/* passing a JS Date */}
-  <DateTime date={new Date()} />
-  {/* passing a Unix Epoch in milliseconds */}
-  <DateTime date={1629240300000} />
-</Stack>
+### Example with passing a Timestamp as an ISOString
```

Comment:
I'd omit `Example with...` part and just name "Passing a timestamp as ISOString"